lib/src/effects/combined_effect.dart

```diff
@@ -1,4 +1,5 @@
 import 'dart:math';
+import 'dart:ui';
```

Comment:
TIL this comes from `dart:ui`, I have always thought it came from widget.dart, interesting

Reply:
it is exported in many places, I chose dart ui because its the one that fit the best with the effects